### PR TITLE
Updated documentation to include expected payload type.

### PR DESCRIPTION
**Pull request by cppwfs**
Relates to spring-cloud-stream-app-starters/app-starters-release#99

### CODE REVIEW

File: spring-cloud-starter-stream-sink-task-launcher-local/README.adoc [hunk 1]
```diff
@@ -6,12 +6,25 @@ in the message that is received.
...
 
+== Input
+
+=== Headers
+
+N/A
+
+=== Payload
+
+* `org.springframework.cloud.task.launcher.TaskLaunchRequest`
```
Comment (artembilan) [inline spring-cloud-starter-stream-sink-task-launcher-local/README.adoc]: Is there any confirmation that this one can be (de)serialized to/from JSON?
Doesn't look persuasively to use this kind of data in between Microservices...

**Review by artembilan — CHANGES_REQUESTED** [review]

File: spring-cloud-starter-stream-sink-task-launcher-local/README.adoc (hunk 1 above)
Comment (cppwfs) [reply spring-cloud-starter-stream-sink-task-launcher-local/README.adoc]: We have tested the task-launchers using triggertask or  transformers which produce TaskLaunchRequest objects.

Comment (artembilan) [reply spring-cloud-starter-stream-sink-task-launcher-local/README.adoc]: Good. Was there a test when this `TaskLaunchRequest` is sent/received as JSON or only serialization variants?

**Comment by sobychacko** [comment]
Merging as the review comments are addressed.

**Pull request closed without merging**